core/include/cubos/core/al/audio_device.hpp

```diff
@@ -90,14 +86,11 @@ namespace cubos::core::al
         class CUBOS_CORE_API Buffer
         {
         public:
-            virtual ~Buffer() = default;
+            ma_decoder mDecoder;
```

Comment:
This file is meant to abstract miniaudio, so there should be no references at all to it here.